compiler/rustc_attr_parsing/src/session_diagnostics.rs

```diff
@@ -571,24 +571,24 @@ pub(crate) enum AttributeParseErrorReason {
     ExpectedNameValue(Option<Symbol>),
     DuplicateKey(Symbol),
     ExpectedSpecificArgument {
-        possibilities: Vec<&'static str>,
+        possibilities: &'a [Symbol],
```

Comment:
if you really wanted to avoid introducing the lifetime, these are seemingly always `&'static [Symbol]`s, right?

Reply:
Sounds good. r=me when green, unless you really wanted lcnr's r+ ofc